src/AddressRegistry.Api.Oslo/AddressMatch/Responses/AddressMatchOsloCollection.cs

```diff
@@ -22,6 +22,13 @@ namespace AddressRegistry.Api.Oslo.AddressMatch.Responses
     [DataContract(Name = "AdresMatchCollectie", Namespace = "")]
     public class AddressMatchOsloCollection
     {
+        /// <summary>
+        /// De linked-data context van het adres.
+        /// </summary>
+        [DataMember(Name = "@context", Order = 0)]
+        [JsonProperty(Required = Required.DisallowNull)]
+        public string Context { get; set; }
```

Comment:
@ArneD I think `Context` should live in the root of the response and `Type` on an `AddressMatchOsloItem`. But I'm not sure the value of `Type` is correct, see https://github.com/Informatievlaanderen/address-registry/blob/main/src/AddressRegistry.Api.Oslo/AddressMatch/Responses/AddressMatchOsloCollection.cs#L54-L59 ?

Reply:
Correct, Type should be on the item.